fuse-products/src/main/java/software/tnb/product/csb/application/LocalSpringBootApp.java

```diff
@@ -56,8 +56,8 @@ public LocalSpringBootApp(AbstractIntegrationBuilder<?> integrationBuilder) {
            command.add("-javaagent:" + projectPath.resolve("src/main/resources/").resolve(integrationBuilder.getJvmAgentPath()).toAbsolutePath());
         }
 
-        command.addAll(args);
         command.add("-jar");
+        command.addAll(args);
         fileName = integrationTarget.resolve(jarName).toAbsolutePath().toString();
```

Comment:
:thinking: are you sure this is valid?

the command should be `java [options] -jar jarfile [args ...]` and this change would make it `java -jar [options] jarfile [args ...]`

Reply:
actually, both works, but let's leave as it is, thanks!